src/NzbDrone.Api/Queue/QueueResource.cs

```diff
@@ -17,7 +19,10 @@ public class QueueResource : RestResource
         public decimal Size { get; set; }
         public string Title { get; set; }
         public decimal Sizeleft { get; set; }
+
+        [JsonConverter(typeof(TimeSpanConverter))]
         public TimeSpan? Timeleft { get; set; }
```

Comment:
moment.js FTW:

`As of 2.1.0, moment supports parsing ASP.NET style time spans.` We'll send it down and let the UI figure it out.